Source/JavaScriptCore/wasm/WasmOMGIRGenerator.cpp

```diff
@@ -859,10 +859,11 @@ class OMGIRGenerator {
     void reloadMemoryRegistersFromInstance(const MemoryInformation&, Value* instance, BasicBlock*);
 
     Value* loadFromScratchBuffer(unsigned& indexInBuffer, Value* pointer, B3::Type);
-    void connectControlAtEntrypoint(unsigned& indexInBuffer, Value* pointer, ControlData&, Stack& expressionStack, ControlData& currentData, bool fillLoopPhis = false);
+    void connectControlAtEntrypoint(unsigned& indexInBuffer, Value* pointer, ControlData&, Stack& expressionStack, bool loadExceptionVar, bool fillLoopPhis = false);
```

Comment:
I guess a WebKit style nit would be to make this an `enum class LoadExceptionVar { Yes, No } ` rather than a bool. It's hard to know what these bools mean sometimes otherwise.